pom.xml

```diff
@@ -274,6 +275,13 @@
 			<type>jar</type>
 			<scope>provided</scope>
 		</dependency>
+		
+		<dependency>
+				<groupId>org.openmrs.module</groupId>
+				<artifactId>emrapi-condition-list</artifactId>
+				<version>${emrApiVersion}</version>
+				<scope>provided</scope>
+		</dependency>
```

Comment:
I'm not sure what the intention is here.  We do not want the reporting module to depend on emrapi as far as I know.  And I would expect that we can't add a dependency on the jar without adding a dependency on the module in config.xml.

Reply:
on using the SQLQueryBuilder rather than the HQLQueryBuilder , i now dont need this dependence, i just forgot to remove it